src/applications/vaos/routes.jsx

```diff
@@ -33,7 +33,7 @@ export default function createRoutesWithStore(store) {
     '/new-covid-19-vaccine-appointment',
     '/schedule/covid-vaccine',
   ];
-  const communityCarePaths = ['/schedule-referral'];
+  const communityCarePaths = ['/schedule-referral', '/complete/:id'];
```

Comment:
I feel like we should talk about this path with UX. I would rather keep it within `/schedule-referral` not sure why they broke it out in figma. Seems like this will potentially have collisions with other apps or features

Reply:
ah okay cool, let me know if you think we should have a larger conversation about it in a PL or something.